src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -252,6 +254,14 @@ def verify_user_emails(
             results[user_id] = RpcVerifyUserEmail(email=email, exists=exists)
         return results
 
+    def get_user_avatar(self, *, user_id: int) -> RpcAvatar | None:
+        possible_avatar = UserAvatar.objects.filter(user_id=user_id).first()
+
+        if not possible_avatar:
+            return None
+
+        return serialize_user_avatar(avatar=possible_avatar)
```

Comment:
Should this be in these changes?

Reply:
I was planning to rebase this after the parent landed, otherwise this would have just failed out of CI